### DISCUSSION TITLE
Add a sandbox export

### DISCUSSION THREAD

**Discussion by Foosec** [sandboxie-plus/Sandboxie#1501] [Various ideas and suggestions] [answered]
Having the ability to just export an entire sandbox (contents and settings) and import it after an OS reinstall, would greatly speed up migrations. (If you have the software actually installed in the sandbox and not just force running there).

Also, perhaps a seperate repository for sandbox config files, so we can share configs for locking down software to whats needed.

**Comment by isaak654** [comment, marked as answer]
> Having the ability to just export an entire sandbox (contents and settings) and import it after an OS reinstall, would greatly speed up migrations. (If you have the software actually installed in the sandbox and not just force running there).

It was introduced in Sandboxie Plus starting with v1.6.0.

> Also, perhaps a seperate repository for sandbox config files, so we can share configs for locking down software to whats needed.

#2768
